test/server/lib/HostedStorageManager.ts

```diff
@@ -439,6 +450,7 @@ describe('HostedStorageManager', function() {
       });
 
       afterEach(async function() {
+        delete process.env.GRIST_SKIP_REDIS_CHECKSUM_MISMATCH;
```

Comment:
Can you grep other tests for use of `EnvironmentSnapshot`? When modifying the environment, we've slowly almost-standardized on saving it in before() and then doing a restore after. Probably won't matter in this specific case but env var changes somewhere in tests can be very hard to debug when they interact.